api/goldens/aptos_api__tests__state_test__test_get_account_resource.json

```diff
@@ -11,7 +11,7 @@
         }
       }
     },
-    "guid_creation_num": "4",
+    "guid_creation_num": "2",
```

Comment:
So, we're getting a breaking change on the GUID of events?

Reply:
coin register event is never the first event to register so this happens.